lib/models/food.js

```diff
@@ -56,17 +53,72 @@ class Food {
       })
     }
 
-    static defineEventHandlers() {
-      $('.form-food').on('submit', (event) => {
-        Food.addNewFood(event);
-      });
+    static newFoodEvent(){
+        $('.form-food').on('submit', (event) => {
+            Food.addNewFood(event);
+        });
+    }
 
-      $('.all-foods').on('click', '.delete', (event) => {
-        const id = event.target.dataset.id;
-        Food.delete(id);
-      });
+    static deleteEvent(){
```

Comment:
@chantal66 I like these clearly-named functions for creating, updating, and deleting foods. This is a major improvement over our previous setup of using a single defineEventHandlers() function which did lots of different things. This refactor makes everything much more explicit and readable. Nice!